Core/GameInstanceManager.cs

```diff
@@ -608,14 +608,14 @@ public static bool IsGameInstanceDir(DirectoryInfo path)
         /// </summary>
         /// <param name="path">A DirectoryInfo of the path to check</param>
         /// <param name="user">IUser object for interaction</param>
-        /// <returns>An instance of the matching game, or null if none could be found</returns>
+        /// <returns>An instance of the matching game, or null if the user cancelled, or throws NotKSPDirKraken if no games found</returns>
```

Comment:
C# uses `exception` tags to document possible exceptions.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#recommended-tags
Like here:
https://github.com/KSP-CKAN/CKAN/blob/2f55a1e67801f05a5fd612de2db5a4db29e23862/Core/GameInstanceManager.cs#L220-L224

Reply:
OK, changed to use that syntax. :+1: